.pre-commit-config.yaml

```diff
@@ -51,7 +51,7 @@ repos:
         entry: --entrypoint /bin/hadolint hadolint/hadolint:latest -
         exclude: ^dev/.*$
   - repo: git://github.com/pre-commit/pre-commit-hooks
-    sha: v1.2.3
+    rev: v1.2.3
```

Comment:
I don't use the precommit hooks myself, does someone want to chime in on this change? (@xhochy?) It looks correct to me, though.

Reply:
I am going to remove this line, it snuck in accidentally. I will have a separate PR for `.pre-commit-config.yaml` landing later.

Reply:
It's correct and would be nice to have as a separate PR